common-lib/src/cmr/common/api/errors.clj

```diff
@@ -66,33 +64,46 @@
   [error]
   (update-in error [:path] keyword-path->string-path))
 
-(defmethod errors->body-string mt/json
-  [_ errors]
-  (json/generate-string {:errors (map error->json-element errors)}))
+;; *************************************
```

Comment:
why has ;; **** ... been added?

Reply:
Looking at the full file I now see why.
Can you just convert the three functions into 1 and use a case statement? It will execute much faster that way.  defmulti/defmethods are not needed here.

Reply:
I"m only doing this when I find there are more then one defmulti in a file and it seams hard to tell them apart.